### ISSUE TITLE
Vulnerability and fix for jackson-databind Java package isn't reported correctly

### ISSUE THREAD

**Issue by thehh1974** [anchore/anchore-engine#1161]
**Is this a BUG REPORT or a FEATURE REQUEST?** (choose one):
BUG REPORT

**Version of Anchore Engine and Anchore CLI if applicable**:
0.9.1

**What happened**:
Scanned image wurstmeister/kafka:2.12-2.5.0 got the following vulnerability:
Vulnerability ID        Package                        Severity        **Fix**         CVE Refs              Vulnerability URL                                      Type        Feed Group        Package Path
CVE-2020-25649          jackson-databind-2.10.2        High            **None**        CVE-2020-25649        https://nvd.nist.gov/vuln/detail/CVE-2020-25649        java        nvdv2:cves        /opt/kafka_2.12-2.5.0/libs/jackson-databind-2.10.2.jar

The vulnerability is reported both by NVD and by GHSA as fixed in version 2.10.5.1.

**What did you expect to happen**:
I expected the GHSA to be reported as well as the fix version.

**Any relevant log output from /var/log/anchore**:
No

**What docker images are you using**:
anchore/anchore-engine:v0.9.1

**How to reproduce the issue**:
Scan wurstmeister/kafka:2.12-2.5.0

**Anything else we need to know**:
No

**Comment by zburstein** [comment]
Hello @thehh1974. I just ran some testing on this and identified that this issue is resolved in the most recent release of engine v0.10.1. I would recommend updating

```
Vulnerability ID           Package                                Severity        Fix                     CVE Refs              Vulnerability URL                                                   Type        Feed Group         Package Path
GHSA-288c-cq4h-88gq        jackson-databind-2.10.2                High            2.10.5.1                CVE-2020-25649        https://github.com/advisories/GHSA-288c-cq4h-88gq                   java        github:java        /opt/kafka_2.12-2.5.0/libs/jackson-databind-2.10.2.jar
```

**Comment by thehh1974** [comment]
Thanks @zburstein! Any idea whether it is fixed in 0.9.4? Or which commit fixed it?
I'm trying to determine the path to fixing this issue with the least side effects.

**Comment by zburstein** [comment]
@thehh1974 Consulting the [release notes](https://engine.anchore.io/docs/releasenotes/093/) I believe that it was fixed as a part of the changes to address https://github.com/anchore/anchore-engine/issues/950, which resolves some issues with the way GHSA's match to Java packages. This was released as a part of v0.9.3
